docs/examples/tmp/imglog0001-1needle-shape_blue_circle.match

```diff
@@ -0,0 +1,12 @@
+# IMAGE MATCH DATA
+[type]
+backend = cv
+
+[find]
+backend = template
+similarity = <value='0.8' min='0.0' max='1.0' delta='10.0' tolerance='1.0' fixed='True' enumerated='False'>
+
+[template]
+backend = ccoeff_normed
+nocolor = <value='False' min='None' max='None' delta='10.0' tolerance='1.0' fixed='True' enumerated='False'>
+
```

Comment:
So this file should be removed independently of whether we ignore future ones or not

Reply:
Same for the ones that follow it